src/odemis/gui/cont/tabs.py

```diff
@@ -2429,36 +2477,69 @@ def _enable_movement_controls(self, cancelled=False):
         """
-        # The move button should turn green only if current move is known and not cancelled
-        if current_position in self.position_btns.keys() and not cancelled:
-            currently_pressed = self.position_btns[current_position]
-            self._toggle_switch_buttons(currently_pressed)
-            currently_pressed.icon_on = img.getBitmap(self.btn_toggle_icons[currently_pressed][1])
-            currently_pressed.Refresh()
-        else:
-            self._toggle_switch_buttons(currently_pressed=None)
-        # Define which button to disable in respect to the current move
-        disable_buttons = {LOADING: MILLING, IMAGING: None, MILLING: COATING, COATING: MILLING, LOADING_PATH: MILLING}
-        for movement, button in self.position_btns.items():
-            if current_position == UNKNOWN:
-                # Only enable loading button when the move is unknown
-                button.Enable() if movement == LOADING else button.Disable()
-            elif movement == disable_buttons[current_position]:
-                button.Disable()
+        if self.tab_data_model.main.role == 'enzel':
+            # The move button should turn green only if current move is known and not cancelled
+            if current_position in self.position_btns.keys() and not cancelled:
+                currently_pressed = self.position_btns[current_position]
+                self._toggle_switch_buttons(currently_pressed)
+                currently_pressed.icon_on = img.getBitmap(self.btn_toggle_icons[currently_pressed][1])
+                currently_pressed.Refresh()
             else:
-                button.Enable()
+                self._toggle_switch_buttons(currently_pressed=None)
+            # Define which button to disable in respect to the current move
+            disable_buttons = {LOADING: MILLING, IMAGING: None, MILLING: COATING, COATING: MILLING, LOADING_PATH: MILLING}
+            for movement, button in self.position_btns.items():
+                if current_position == UNKNOWN:
+                    # Only enable loading button when the move is unknown
+                    button.Enable() if movement == LOADING else button.Disable()
+                elif movement == disable_buttons[current_position]:
+                    button.Disable()
+                else:
+                    button.Enable()
+        elif self.tab_data_model.main.role == 'meteor':
+            # The move button should turn green only if current move is known and not cancelled
+            if current_position in self.position_btns.keys() and not cancelled:
+                currently_pressed = self.position_btns[current_position]
+                self._toggle_switch_buttons(currently_pressed)
+                currently_pressed.icon_on = img.getBitmap(self.btn_toggle_icons[currently_pressed][1])
+                currently_pressed.Refresh()
+            else:
+                self._toggle_switch_buttons(currently_pressed=None)
```

Comment:
That part is identical to enzel -> put it before the check..

Reply:
Not anymore actually. Once i push, you will see it got different.